themes/classic/_dev/js/theme.js

```diff
@@ -12,9 +12,10 @@ import './product';
 import './cart';
 
 import DropDown from './components/drop-down';
-import TopMenu from './components/top-menu';
+import Form from './components/form';
```

Comment:
Just an idea not really linked to your pull request:

Why not create an index.js on components and use:

```js
import { Form, TopMenu } fom './components'
```